packages/framework-integration-tests/integration/providers/kubernetes/deployment/deployment.integration.ts

```diff
@@ -1,6 +1,7 @@
 import { expect } from '../../../helper/expect'
 import * as Kubernetes from '@kubernetes/client-node'
 import { boosterKubernetesServices, kubernetesNamespace } from '../constants'
+import { waitForIt } from '../../aws/utils'
```

Comment:
should we move this method to a generic `utils` file?

Reply:
I think so, since it's used in other providers than aws

Reply:
It is already in the generic `helpers` folder. It was duplicated.

Reply:
Also, I removed the whole `utils.ts` file in PR https://github.com/boostercloud/booster/pull/817

Another point of that PR is that, when it gets merged, you can create some little implementation of an interface and run the integration test for k8s too (although I'm not sure if all of them would pass, as it doesn't have subscriptions, for example)

Reply:
awesome! I changed it to the `helpers` one 😄